modules/wmco-prerequisites.adoc

```diff
@@ -30,6 +30,9 @@ a|* Windows Server 2022, OS Build link:https://support.microsoft.com/en-us/topic
 |Google Cloud Platform (GCP)
 |Windows Server 2022, OS Build link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[20348.681] or later
 
+|Nutanix
```

Comment:
Not critical, but this table might look better alphabetized. Same with later in the file